engine/Shopware/Controllers/Backend/Base.php

```diff
@@ -1043,15 +1043,20 @@ public function validateEmailAction()
         $this->Front()->Plugins()->ViewRenderer()->setNoRender();
         $this->Front()->Plugins()->Json()->setRenderer(false);
 
-        $email = $this->Request()->getParam('value');
+        $emails = explode(',', $this->Request()->getParam('value', ''));
```

Comment:
```suggestion
        $emails = explode(',', (string) $this->Request()->getParam('value', ''));
```

just to make sure, explode gets always a string. I think it would otherwise give some notices on newer PHP versions :thinking: